charts/dependency-track/values.yaml

```diff
@@ -10,6 +10,11 @@ common:
     createSecret: false
     # -- Use the secret key defined in an existing secret.
     existingSecretName: ""
+  serviceAccount:
+    create: create
```

Comment:
Should this be a boolean instead?

Reply:
Definitely. I'll make the change tomorrow